### PR TITLE
Clarified the need to call scene.render in between picks with pickTranslucentDepth

### PR DESCRIPTION
**Pull request by hanbollar**
Fixes [#6151](https://github.com/AnalyticalGraphicsInc/cesium/issues/6151)

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Signed CLA is on file.

@hanbollar, thanks for the pull request! Maintainers, we have a signed CLA from @hanbollar, so you can review this at any time.

:warning: I noticed that [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CHANGES.md) has not been updated. If this change updates the public API in any way, fixes a bug, or makes any non-trivial update, please add a bullet point to `CHANGES.md` and comment on this pull request so we know it was updated. For more info, see the [Pull Request Guidelines]( https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines).

---

__I am a bot who helps you make Cesium awesome! [Contributions to my configuration are welcome.](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/.concierge)__

:earth_africa: :earth_americas: :earth_asia:

**Comment by hanbollar** [comment]
@ggetz lmk if this is the update you're looking for

File: Source/Scene/Scene.js [hunk 1]
```diff
@@ -626,12 +626,26 @@ define([
...
+         * @example
+         * // picking the position of a translucent primitive
+         * viewer.screenSpaceEventHandler.setInputAction(function onLeftClick(movement) {
+         *      var pickedFeature = viewer.scene.pick(movement.position);
+         *      if (!Cesium.defined(pickedFeature)) {
+         *          // nothing picked
+         *          return;
+         *      }
+         *      viewer.scene.render();
+         *      console.log(viewer.scene.pickPosition(movement.position));
```
Comment (hpinkos) [inline Source/Scene/Scene.js]: @hanbollar change this to something like `var worldPosition = viewer.scene.pickPosition(movement.position));`

**Comment by hanbollar** [comment]
@hpinkos updated with the change

**Comment by hpinkos** [comment]
Great, thanks!

**Pull request merged**